lib/nerves/artifact/build_runners/docker.ex

```diff
@@ -201,6 +211,39 @@ defmodule Nerves.Artifact.BuildRunners.Docker do
     end
   end
 
+  defp set_volume_permissions(pkg) do
+    {_dockerfile, image} = config(pkg)
+
+    args =
```

Comment:
I'm good with this if you add a comment here to explain what's going on with the args and `--rm` and `chown`.